app/javascript/components/widgets/twitter/tweet_content.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Microlink from '@microlink/react';
+import { colors, fonts } from '../../../lib/theme';
+
+const ContentWrapper = styled.div`
+  display: block;
+  width: auto;
+  height: 487px;
+  margin-bottom: 29px;
+`;
+
+const ContentImage = styled.img`
+  display: block;
+  width: auto;
+  height: 487px;
+  border-radius: 6px;
+  border-collapse: separate;
+`;
+
+const TweetContent = ({ mediaUrl, linkUrl }) => (
+  <ContentWrapper>
+    {mediaUrl !== null ? <ContentImage src={mediaUrl} alt="twitter" /> : null}
+    {linkUrl !== null && mediaUrl === null ? (
+      <Microlink
+        url={linkUrl}
+        style={{
+          fontFamily: `${fonts.regular}`,
+          height: '487px',
+          maxWidth: '100%',
+          color: `${colors.white}`,
+          backgroundColor: `${colors.black}`,
+          borderRadius: '6px',
+          borderCollapse: 'separate',
+          border: 'solid 1px #333333',
```

Comment:
Same here. Could this be resolved with a containing styled component? Or by children / a component prop?

Reply:
I tried using styled-components on the Microlink and for some reason the API wouldn't accept the styling (it was in the docS). I'm not super familiar on how to style API components with styled so I'm not sure if it was me or whatever